File: app/src/main/java/com/geridea/trentastico/gui/fragments/AboutFragment.java
package com.geridea.trentastico.gui.fragments;


/*
 * Created with ♥ by Slava on 22/04/2017.
 */

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.MenuItem;
import android.view.View;
import android.view.ViewGroup;
import android.widget.FrameLayout;
import android.widget.TextView;

import com.geridea.trentastico.BuildConfig;
import com.geridea.trentastico.R;
import com.geridea.trentastico.gui.activities.FragmentWithMenuItems;
import com.geridea.trentastico.utils.DebugUtils;
import com.github.porokoro.paperboy.PaperboyFragment;

import butterknife.BindView;
import butterknife.ButterKnife;

public class AboutFragment extends FragmentWithMenuItems {

    @BindView(R.id.version_text) TextView versionText;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_about, container, false);
        ButterKnife.bind(this, view);

        versionText.setText(DebugUtils.computeVersionName());

        PaperboyFragment paperboyFragment = PaperboyFragmentMaker.buildPaperboyFragment(getContext());
        getChildFragmentManager()
            .beginTransaction()
            .replace(R.id.paperboy_fragment_frame, paperboyFragment)
            .commit();

        return view;
    }

    @Override
    public int[] getIdsOfMenuItemsToMakeVisible() {
        return new int[0];
    }

    @Override
    public void bindMenuItem(MenuItem item) {
        //No menus used
    }

}

File: app/src/main/java/com/geridea/trentastico/gui/fragments/AboutFragment.kt
package com.geridea.trentastico.gui.fragments


/*
 * Created with ♥ by Slava on 22/04/2017.
 */

import android.app.AlertDialog
import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.MenuItem
import android.view.View
import android.view.ViewGroup
import android.widget.Toast
import com.alexvasilkov.android.commons.utils.Views
import com.geridea.trentastico.R
import com.geridea.trentastico.gui.activities.FragmentWithMenuItems
import com.geridea.trentastico.utils.AppPreferences
import com.geridea.trentastico.utils.DebugUtils
import kotlinx.android.synthetic.main.dialog_licenes.view.*
import kotlinx.android.synthetic.main.fragment_about.view.*

class AboutFragment : FragmentWithMenuItems() {

    var debugClickCounter = 0

    override fun onCreateView(
            inflater: LayoutInflater,
            container: ViewGroup?,
            savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_about, container, false)

        view.versionText.text = DebugUtils.computeVersionName()

        view.licencesBtn.setOnClickListener {
            LicencesDialog(requireContext()).show()
        }

        return view
    }

    override val idsOfMenuItemsToMakeVisible: IntArray
        get() = IntArray(0)

    override fun bindMenuItem(item: MenuItem) = /*No menus used*/ Unit

}


class LicencesDialog(context: Context) : AlertDialog(context) {

    init {
        val view = Views.inflate<View>(context, R.layout.dialog_licenes)
        view.licencesTV.text = context.getText(R.string.licence)

        setView(view)
    }

}

